### PR TITLE
Droth 808 tr to oth line assets with location

### PR DESCRIPTION
**Pull request by MAC-CGI**
Droth 808 tr to oth line assets with location

### CODE REVIEW

**Comment by joanacoelho** [comment]
ok

File: digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/util/TierekisteriImporterOperations.scala [hunk 1]
```diff
@@ -382,5 +390,119 @@ class RoadWidthTierekisteriImporter extends LinearAssetTierekisteriImporterOpera
...
+  protected override def getAllTierekisteriHistoryAddressSection(roadNumber: Long, lastExecution: DateTime) = {
+    println("\nFetch " + assetName + " History by Road Number " + roadNumber)
+    val trAsset = tierekisteriClient.fetchHistoryAssetData(roadNumber, Some(lastExecution)).filter(_.laneType != TRLaneArrangementType.Unknown)
+
+    trAsset.foreach { tr => println(s"TR: address ${tr.roadNumber}/${tr.startRoadPartNumber}-${tr.endRoadPartNumber}/${tr.track.value}/${tr.startAddressMValue}-${tr.endAddressMValue}") }
+    trAsset.map(_.asInstanceOf[TierekisteriAssetData]).flatMap(getRoadAddressSections)
+  }
+
+  override protected def createLinearAsset(vvhRoadlink: VVHRoadlink, roadAddress: ViiteRoadAddress, section: AddressSection, measures: Measures, trAssetData: TierekisteriAssetData): Unit = {
+    if (trAssetData.laneType == MassTransitLane) {
```
Comment (MAC-CGI) [inline digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/util/TierekisteriImporterOperations.scala]: This line was replaced by the filter(_.laneType != TRLaneArrangementType.Unknown), could be deleted

File: digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/util/TierekisteriImporterOperationsSpec.scala [hunk 2]
```diff
@@ -374,14 +496,179 @@ class TierekisteriImporterOperationsSpec extends FunSuite with Matchers  {
...
+      val assetI = linearAssetDao.fetchLinearAssetsByLinkIds(testPavedRoad.typeId, Seq(5001), LinearAssetTypes.numericValuePropertyId).head
+
+      testPavedRoad.updateAssets(DateTime.now())
+      val assetU = linearAssetDao.fetchLinearAssetsByLinkIds(testPavedRoad.typeId, Seq(5001), LinearAssetTypes.numericValuePropertyId).filterNot(a => a.id == assetI.id).head
+
+      assetU.startMeasure should not be assetI.startMeasure
+      assetU.endMeasure should be (assetI.endMeasure)
+    }
+  }
+
```
Comment (MAC-CGI) [inline digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/util/TierekisteriImporterOperationsSpec.scala]: maybe we don't need these tests, the import and the updates are equals

**Comment by leirasf** [comment]
Check comments on the pull request of [DROTH-809](https://github.com/finnishtransportagency/digiroad2/pull/860)

**Comment by marquesfm** [comment]
ok

**Pull request merged**